src/Akka.Persistence.Azure/Journal/AzureTableStorageJournal.cs

```diff
@@ -383,15 +383,19 @@ protected override void PreStart()
                             if (_log.IsDebugEnabled && _settings.VerboseLogging)
                                 foreach (var r in persistenceResults)
                                     _log.Debug("Azure table storage wrote entity [{0}] with status code [{1}]", r.Etag, r.HttpStatusCode);
+
+                            exceptions = exceptions.Add(null);
```

Comment:
This looks correct - we do the same thing in Akka.Persistence.SqlServer:

https://github.com/akkadotnet/akka.net/blob/ceefa2805b2ef1e5760255275abc149f280cc5a0/src/contrib/persistence/Akka.Persistence.Sql.Common/Journal/SqlJournal.cs#L174-L177